specification/billing/resource-manager/Microsoft.Billing/preview/2018-11-01-preview/billing.json

```diff
@@ -0,0 +1,803 @@
+            "schema": {
+              "$ref": "#/definitions/Invoice"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.Billing/invoices/latest": {
+      "get": {
+        "tags": [
+          "Invoices"
+        ],
+        "operationId": "Invoices_GetLatest",
+        "description": "Gets the most recent invoice. When getting a single invoice, the downloadUrl property is expanded automatically.  This is only supported for Azure Web-Direct subscriptions. Other subscription types which were not purchased directly through the Azure web portal are not supported through this preview API.",
+        "x-ms-examples": {
+          "InvoicesGetLatest": {
+            "$ref": "./examples/InvoicesGetLatest.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/subscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK. The request has succeeded.",
+            "schema": {
+              "$ref": "#/definitions/Invoice"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        }
+      }
+    },
+    "/providers/Microsoft.Billing/billingAccounts/{billingAccountId}/invoiceSections/{invoiceSectionId}/products/{productName}/transfer": {
+      "post": {
+        "tags": [
+          "Products"
+        ],
+        "operationId": "Products_Transfer",
+        "description": "The operation to transfer a Product to another InvoiceSection.",
+        "x-ms-examples": {
+          "TransferProduct": {
+            "$ref": "./examples/TransferProduct.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/billingAccountIdParameter"
+          },
+          {
+            "$ref": "#/parameters/invoiceSectionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/productNameParameter"
+          },
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          },
+          {
+            "name": "parameters",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/TransferProductProperties"
+            },
+            "description": "Parameters supplied to the Transfer Product operation."
+          }
+        ],
+        "responses": {
+          "202": {
+            "description": "Accepted",
+            "headers": {

```

Comment:
Which header contains the async operation status URI? Location or Azure-AsyncOperation? I think you can also simplify this to something like [this](https://github.com/Azure/azure-rest-api-specs/blob/105dafa9796a97856db761e5c7518872d6075398/specification/compute/resource-manager/Microsoft.Compute/stable/2018-10-01/compute.json#L4284-L4298)

Reply:
I am following the pattern in our swagger so far, location is the operationsresults and asyncoperations - for the async operation status. We have internally debated on this and the next PR with all the changes, we will be working to address that. I will be looping in our lead developer separately.